src/test/java/com/softserveinc/edu/ita/tests/item_management_page/AddNewProductTest.java

```diff
@@ -0,0 +1,111 @@
+
+import static com.softserveinc.edu.ita.utils.DBUtility.getLastAddedProduct;
+import static com.softserveinc.edu.ita.utils.DBUtility.removeProductFromDatabase;
+
+public class AddNewProductTest extends TestRunner {
+
+    private HomePage homePage;
+    private UserInfoPage userInfoPage;
+    private ItemManagementPage itemManagementPage;
+    private AddProductPage addProductPage;
+
+    @Test(dataProvider = "getSupervisors", dataProviderClass = DataProviders.class)
+    public void cancelProductAddingTest(User user) {
+
+        homePage = new HomePage(driver);
+        userInfoPage = homePage.logIn(user.getLogin(), user.getPassword());
+        itemManagementPage = userInfoPage.clickItemManagementTab();
+
+        final int numberOfFoundProducts = itemManagementPage.getFoundProductsNumber();
+
+        addProductPage = itemManagementPage.clickAddProductLink();
+
+        loggingSoftAssert.assertTrue(addProductPage.getProductNameFieldText().isEmpty(),
+                "<b>Product name field</b> is empty");
+        loggingSoftAssert.assertTrue(addProductPage.getProductDescriptionFieldText().isEmpty(),
+                "<b>Product description field</b> is empty");
+        loggingSoftAssert.assertTrue(addProductPage.getProductPriceFieldText().isEmpty(),
+                "<b>Product price field</b> is empty");
+
+        final Product newProduct = addProductPage.createRandomProduct();
+
+        itemManagementPage = addProductPage.fillProductName(newProduct.getProductName())
+                .fillProductDescription(newProduct.getProductDescription())
+                .fillProductPrice(newProduct.getProductPrice())
+                .clickCancelButton();
+
+        loggingAssert.assertEquals(numberOfFoundProducts, itemManagementPage.getFoundProductsNumber(),
+                String.format("Number of products didn't change: expected - <b>%s</b>; actual - <b>%s</b>;",
+                        numberOfFoundProducts, itemManagementPage.getFoundProductsNumber()));
+
+        loggingSoftAssert.assertAll();
+    }
+
+    @Test(dataProvider = "getSupervisors", dataProviderClass = DataProviders.class)
+    public void addNewProductTest(User user) {
+
+        homePage = new HomePage(driver);
+        userInfoPage = homePage.logIn(user.getLogin(), user.getPassword());
+        itemManagementPage = userInfoPage.clickItemManagementTab();
+
+        final int numberOfFoundProducts = itemManagementPage.getFoundProductsNumber();
+
+        addProductPage = itemManagementPage.clickAddProductLink();
+
+        final Product newProduct = addProductPage.createRandomProduct();
+
+        itemManagementPage = addProductPage.fillProductName(newProduct.getProductName())
+                .fillProductDescription(newProduct.getProductDescription())
+                .fillProductPrice(newProduct.getProductPrice())
+                .clickOkButton();
+
+        loggingAssert.assertEquals(newProduct.getProductName(), getLastAddedProduct().getProductName(),
```

Comment:
getLastAddedProduct() from what? do not static import DBUtility, write DBUtility.getLastAddedProduct(), that makes it more obvious, from where the product is